lzero/mcts/buffer/game_buffer_stochastic_muzero.py

```diff
@@ -85,7 +85,7 @@ def _make_batch(self, batch_size: int, reanalyze_ratio: float) -> Tuple[Any]:
             ]
             if self._cfg.use_ture_chance_label_in_chance_encoder:
                 chances_tmp += [
-                    np.random.randint(0, game.action_space_size)
+                    np.random.randint(0, self._cfg.model.chance_space_size)
```

Comment:
Hello, could you please confirm if you have already tested this modification locally and ensured that there are no issues? If so, please rename the PR to match our existing format, and it can be merged. Thank you for your contribution!

Reply:
> Hello, could you please confirm if you have already tested this modification locally and ensured that there are no issues? If so, please rename the PR to match our existing format, and it can be merged. Thank you for your contribution!

Yes I have tested it by running a simple training on a chance based environment. I checked the chance values and they are within the chance space.

Reply:
Updated title

Reply:
Thank you!